php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -245,6 +245,57 @@ class NDB_BVL_Feedback
         return $list;
     }
 
+
+    /**
+     * Get feedback type id given the feedback type name
+     *
+     * @param string $name feedback type name
+     *
+     * @return int feedback type id, empty array if none found
+     */
+    public function getFeedbackTypeIdByName($name)
+    {
+        $factory = NDB_Factory::singleton();
+        $db      = $factory->Database();
+
+        $query = "SELECT  `Feedback_type`
+                    FROM  `feedback_bvl_type`
+                    WHERE  `Name` LIKE  :name ";
```

Comment:
Should this be "=" instead of "LIKE"? I'm not sure what the point is of a LIKE clause that doesn't include a wildcard.